program/rust/src/tests/test_upd_price_v2.rs

```diff
@@ -77,12 +77,11 @@ fn test_upd_price_v2() -> Result<(), Box<dyn std::error::Error>> {
         assert_eq!(price_data.comp_[0].latest_.status_, PC_STATUS_TRADING);
         assert_eq!(price_data.valid_slot_, 0);
```

Comment:
In this file we're cranking for aggregate many times, we don't need to do that anymore so you can make the test file shorter